src/Microsoft.TestPlatform.Build/Tasks/VSTestForwardingApp.cs

```diff
@@ -33,23 +33,15 @@ public int Execute()
                                       FileName = hostExe,
                                       Arguments = string.Join(" ", this.allArgs),
                                       UseShellExecute = false,
-                                      CreateNoWindow = true,
```

Comment:
What are the scenarios we have tried?

* `dotnet test` in cmd, powershell, bash
* `dotnet test` from Process.CreateProcess of any app
* Mimic how `vstest.console` is called from VSTS task or a jenkins/travis or any CI system

We should ensure that CI tasks that run as services (and thus not interactive) don't face any issues with this approach.